controllers/HomeController.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-require_once 'yamf/models/View.php';
+namespace Controllers;
+
+use Yamf\models\Request;
+use Yamf\models\View;
 
 class HomeController {
 
-    public function index($app, $request) {
+    public function index(\stdClass $app, Request $request) {
```

Comment:
We probably don't need to refer to `$app` as a `stdClass`...I don't think that tells the user anything useful. Please remove.

Reply:
An `App` class is something I'll think about. I didn't want to force the user to subclass it just to add extra parameters.